ui/src/components/select/QSelect.json

```
@@ -8,7 +8,7 @@
   "props": {
     "model-value": {
       "extends": "model-value",
-      "type": [ "Number", "String", "Array", "null", "undefined" ],
+      "type": [ "Number", "String", "Array", "Object", "null", "undefined" ],
```

I just figured we are using `Any` in all the other places such as return type of `option-value` prop, `opt` references in scopes of slots, etc. even `@update:model-value` has `Any` implicitly. Also, logically there is nothing holding the users back from using bigint, symbol, etc. too. So, I think we can just simply use the type `Any` here.
```suggestion
      "type": "Any",
```